app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,23 @@
+Invoice id: <%= @invoice.id %><br>
+Invoice status: <%= @invoice.status %><br>
+Invoice created at: <%= @invoice.created_at.strftime('%A, %B %e, %Y') %><br>
+Invoice customer name: <%= @invoice.customer.full_name %><br>
+Invoice total revenue: <%= number_to_currency(@invoice.total_revenue(@invoice.id).fdiv(100)) %><br>
+Inovice items:
+<hr>
+<% @invoice.items.each do |item| %>
+  <div id="item-<%= item.id %>">
+    <% item.invoice_items.each do |inv_item| %>
+      <% if inv_item.invoice_id == @invoice.id %>
+          Item name: <%= item.name %><br>
+          Quantity: <%= inv_item.quantity %><br>
+          Price: <%= number_to_currency(inv_item.unit_price.fdiv(100)) %><br>
+          Status: <%= form_with model: [:admin, inv_item], local: true do |form| %>
+            <%= form.select :status, [["pending"], ["packaged"], ["shipped"]], :selected => inv_item.status %>
```

Comment:
Totally functional--just could save some typing if you desired. No need to put the individual strings in arrays (one overall array would do, and could use the typical hash syntax of `selected: inv_item.status`.